src/Essentials/test/DeviceTests/Tests/WindowStateManager_Tests.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.Maui.ApplicationModel;
+using Microsoft.Maui.Authentication;
+using Xunit;
+
+namespace Microsoft.Maui.Essentials.DeviceTests
+{
+	[Category("WindowStateManager")]
+	public class WindowStateManager_Tests
+	{
+#if __IOS__
+		[Fact]
+		[Trait(Traits.InteractionType, Traits.InteractionTypes.Machine)]
+		public void GetCurrentUIWindow()
+		{
+			var window = WindowStateManager.Default.GetCurrentUIWindow();
+			Assert.NotNull(window);
+		}
+
+		[Fact]
+		[Trait(Traits.InteractionType, Traits.InteractionTypes.Machine)]
+		public void GetCurrentUIViewController()
+		{
+			var windows = WindowStateManager.Default.GetCurrentUIViewController();
+			Assert.NotNull(windows);
+			Assert.NotEmpty(windows);
```

Comment:
I think `GetCurrentUIViewController` returns a single item, so maybe the `NotEmpty` is not correct or a mis-type?

Reply:
I wanted to check the possibility that at least one window came back (not an empty array).  I check to make sure it's always a single item if that makes more sense.

Reply:
But that method returns a single view controller, not a collection of windows.

Reply:
Yup I see now.  Not sure what I was thinking 😕 .  I updated the test to make more sense